app/vep/models/pipeline_model.py

```diff
@@ -12,10 +12,17 @@ class VEPConfigParams(BaseModel):
   vcf: str
   vep_config: str
 
+  @model_serializer
+  def vep_config_serialiser(self):
```

Comment:
This is not needed. You can use [model_dump_json()](https://docs.pydantic.dev/latest/concepts/serialization/#modelmodel_dump_json) instead.

Reply:
@veidenberg  The seqra platform expects stringify encoded json. I tried all combination but couldn't generated it so I have to write that (ugly) code. :(

Reply:
What about `json.dumps(model.model_dump_json())`?

Reply:
This will lead to infinite recursion. :(

Reply:
No it won't, it will just stringify twice, escaping the `"` chars, if that's what nextflow expects.

Reply:
I did try all combination @veidenberg  but it's not working.  The only thing which is working is what we have in the code right now. It just needs paramsText as stringified_encoded_json ( with escaped ") hard to achieve with any combination of json, dumps, loads, model_dump, model_dump_json. 
I tried one more time and now getting 400 as the payload is different. 
```
"{\"launch\":{\"computeEnvId\":\"17wuWKrWOZW5JrLoGGwPdD\",\"pipeline\":\"https://github.com/jonk-ebi/platform-test-pipeline\",\"workDir\":\"/homes/ens_adm02/runs\",\"revision\":\"main\",\"pullLatest\":true,\"configProfiles\":[\"slurmnew\"],\"paramsText\":{\"vcf\":\"/Users/kamal/Development/e2020/cicd/e2s/ensembl-web-tools-api/tmpdy1zmjo9/Venter_chr1.vcf\",\"vep_config\":\"/Users/kamal/Development/e2020/cicd/e2s/ensembl-web-tools-api/tmpdy1zmjo9/config.ini\"},\"preRunScript\":\"module load nextflow\",\"postRunScript\":\"\",\"stubRun\":false,\"labelIds\":[],\"headJobCpus\":1,\"headJobMemoryMb\":4}}"
```